docs/mirror-maintain-guides-how-to-validate-kusama.md

```diff
@@ -82,12 +82,34 @@ command INSTEAD of the previous one:
 brew install cmake pkg-config openssl git llvm
 ```
 
+### Install & Configure Network Time Protocol (NTP) Client
```

Comment:
I don't have a problem with this section but does it belong in this PR?

Reply:
I guess this PR also covers the sync pages, which is fine, just a bigger PR than I was expecting from the title.